tests/manage/test_run_io_multiple_pods.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import logging
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.framework.testlib import ManageTest, tier2
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret, create_pods, create_pvcs
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_rbd_storageclass.__name__,
+    create_pvcs.__name__,
+    create_pods.__name__
+)
+class TestRunIOMultiplePods(ManageTest):
+    """
+    Run IO on multiple pods in parallel
+    """
+    pvc_size = '5Gi'
```

Comment:
let's randomize this size to cover more use cases where there are multiple sizes

Reply:
so in case of randomized size, this won't be executed as part of the regression suite.
Maybe we can create another type of execution for randomized tests